src/frontend/src/pages/Groups/GroupsView.tsx

```diff
@@ -0,0 +1,40 @@
+import { IViewProps } from './types';
+
+import { Grid, Column } from '@carbon/react';
+import { Button } from 'carbon-components-react';
+
+import AppRoutes from 'navigation/AppRoutes';
+import { useNavigate } from 'react-router-dom';
+import { GroupsDataTable } from './GroupsDataTable';
+
+// TODO
+// <Breadcrumb noTrailingSlash>
+//     <BreadcrumbItem>
+//         <a href={AppRoutes.root}>Project</a>
+//     </BreadcrumbItem>
+// </Breadcrumb>
```

Comment:
I'd like to add this as a generic component we can all re-use, but haven't bothered yet